src/main/java/com/autotune/analyzer/recommendations/engine/RecommendationEngine.java

```diff
@@ -67,13 +69,14 @@ public class RecommendationEngine {
     private Map<String, Terms> terms;
     private KruizeObject kruizeObject;
     private Timestamp interval_end_time;
+    private List<String> modelName;
```

Comment:
make it plural since it holds a collection of model names.
Do a refactor so that it updates the corresponding getter and setter methods as well